### DISCUSSION TITLE
Export Matarial Problem

### DISCUSSION THREAD

**Discussion by chlwotjd005** [atteneder/glTFast#467] [Q&A] [answered]
When exporting to version 4.8.3 from the test project, the material or other material properties were well attached to the material, but when I tried to apply it to the main project, the 4.8.5 version was uploaded, and the material properties were not included in the json file as shown below. When imported, properties such as colors are imported while being blown away. I've been trying to solve it alone for 3 days, but I don't know why, so I'm asking you a question. Please reply so that I can solve it.

{"asset":{"version":"2.0","generator":"Unity 2021.3.5f1 glTFast 4.8.5"},"nodes":[{"name":"Export","mesh":0}],"buffers":[{"uri":"OBJ_08376.bin","byteLength":35792}],"bufferViews":[{"buffer":0,"byteLength":4992},{"buffer":0,"byteLength":30800,"byteOffset":4992,"byteStride":56}],"accessors":[{"bufferView":1,"componentType":5126,"count":550,"type":"VEC3","max":[0.5,1,0.5],"min":[-0.5,-1,-0.5]},{"bufferView":1,"componentType":5126,"count":550,"type":"VEC3","byteOffset":12},{"bufferView":1,"componentType":5126,"count":550,"type":"VEC4","byteOffset":24},{"bufferView":1,"componentType":5126,"count":550,"type":"VEC2","byteOffset":40},{"bufferView":1,"componentType":5126,"count":550,"type":"VEC2","byteOffset":48},{"bufferView":0,"componentType":5123,"count":2496,"type":"SCALAR"}]**,"materials":[{"name":"New Material","pbrMetallicRoughness":{"metallicFactor":0}}]**,"meshes":[{"name":"Capsule","primitives":[{"attributes":{"POSITION":0,"NORMAL":1,"TANGENT":2,"TEXCOORD_0":3,"TEXCOORD_1":4},"indices":5,"material":0}]}],"scene":0,"scenes":[{"nodes":[0]}]}

**Comment by atteneder** [comment, marked as answer]
@chlwotjd005 Thanks for reporting,

I can confirm that this is broken on the main branch after the shader refactoring. I'd be surprised if 4.8.5 is affected though. Which version exactly are you running?

Also for the next time: This is clearly an issue and not a discussion.

Thanks
